source/includes/steps-install-shell-base.yaml

```diff
@@ -1,31 +1,28 @@
 ---
-title: "Navigate to the MongoDB Download Center."
+title: "Open the MongoDB Shell download page."
 ref: navigate-dlc
 level: 4
 content: |
 
   Open the `MongoDB Download Center
-  <https://www.mongodb.com/download-center/community?jmp=docs>`__.
+  <https://www.mongodb.com/try/download/shell?jmp=docs>`__.
 ---
-title: "Open the |mdb-shell| Download Page."
-ref: open-mdb-shell-page
-level: 4
-content: |
-
-   Click :guilabel:`Tools`, then select :guilabel:`Shell`.
----
-title: "Download the installation archive for your operating system."
+title: "Download the ``mongosh`` installation archive for your
+  operating system."
```

Comment:
• Thought: you could combine steps 1 & 2 here, like so:

> 1. Download mongosh
> Download the Red Hat 64-bit .rpm package from the <MongoDB Download Center>
>
> 1. Download mongosh
> Download the Debian 64-bit .deb package from the <MongoDB Download Center>
>
> 1. Download mongosh
> Download the Linux 64-bit .tgz package from the <MongoDB Download Center>

Reply:
I'm going to leave this as-is because this change would kind of muck up the single-sourcing for macOS and Windows, and I think this is ok enough where I don't want to spend the time to refactor.